app/models/user_profile.rb

```diff
@@ -150,21 +149,25 @@ def website_domain_validator
 #
 # Table name: user_profiles
 #
-#  user_id              :integer          not null, primary key
-#  location             :string
-#  website              :string
-#  bio_raw              :text
-#  bio_cooked           :text
-#  profile_background   :string(255)
-#  dismissed_banner_key :integer
-#  bio_cooked_version   :integer
-#  badge_granted_title  :boolean          default(FALSE)
-#  card_background      :string(255)
-#  views                :integer          default(0), not null
+#  user_id                      :integer          not null, primary key
+#  location                     :string
+#  website                      :string
+#  bio_raw                      :text
+#  bio_cooked                   :text
+#  profile_background           :string(255)
+#  dismissed_banner_key         :integer
+#  bio_cooked_version           :integer
+#  badge_granted_title          :boolean          default(FALSE)
+#  card_background              :string(255)
+#  views                        :integer          default(0), not null
+#  profile_background_upload_id :integer
+#  card_background_upload_id    :integer
 #
 # Indexes
 #
-#  index_user_profiles_on_bio_cooked_version  (bio_cooked_version)
-#  index_user_profiles_on_card_background     (card_background)
-#  index_user_profiles_on_profile_background  (profile_background)
+#  index_user_profiles_on_bio_cooked_version            (bio_cooked_version)
+#  index_user_profiles_on_card_background               (card_background)
```

Comment:
This index is going no? and the column?

Reply:
@SamSaffron We need to drop the columns in a post migration but I'm playing it safe first by not dropping the columns since we're migrating data.